stix2/pattern_expressions.py

```diff
@@ -0,0 +1,223 @@
+from .constants import Constant, IntegerConstant, ListConstant, make_constant
+from .object_path import ObjectPath
+
+
+class PatternExpression(object):
```

Comment:
Actually, there are a lot of classes that shouldn't be visible - so I changed them to use the "_" notation